docs/resources/files.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Storage"
+---
+# databricks_files Resource
+
+This is a resource that lets you upload and download files upto 2GB in octet-stream.
+
+## Example Usage
+
+In order to manage a file on Databricks File System with Terraform, you must specify the `source` attribute containing the full path to the file on the local filesystem.
```

Comment:
`Databricks File System` ? is it copy-paste? We need to specify to where these files could be uploaded